pgptool-gui/src/main/java/org/pgptool/gui/bkgoperation/UserRequestedCancellationException.java

```diff
@@ -34,6 +34,6 @@ public UserRequestedCancellationException() {
 
 	@Override
 	public String getMessageCode() {
-		return "exception.userRequestedCancelation";
+		return "exception.userRequestedCancellation";
```

Comment:
There should also be corresponding entries in `message*.properties` files.

Reply:
That is a good point... I have not finished reviewing all the code, and have not yet come across the other entries you mentioned... maybe better just to leave as-is, so that nothing will be broken?